app/assets/stylesheets/helpers/_header.scss

```diff
@@ -245,8 +245,21 @@
   @include core-16;
 
   &.visible {
-    @include media(desktop) {
-      display: block;
+    display: block;
+  }
+
+  &.mobile-survey {
```

Comment:
Can we do this without introducing a new class?  The design card (see: https://trello.com/c/jDmt8lpf/146-mobile-survey-design) for this story has versions of both url and email surveys, so I think we are good to let both types of survey appear on mobile (and the CSS changes here are probably enough to cover both designs).

Could we get away with dropping this `&.mobile-survey` rule, and having it's contents apply to the entire `&.visible` block ?